config/galaxy.ini

```diff
@@ -873,7 +1059,7 @@ admin_users = phenomenal@phenomenal-h2020.eu
 # Allow users to remove their datasets from disk immediately (otherwise,
 # datasets will be removed after a time period specified by an administrator in
 # the cleanup scripts run via cron)
-#allow_user_dataset_purge = False
+#allow_user_dataset_purge = True
```

Comment:
We should probably set this to true in our setups, at least for long term running instances.